packages/paste-core/components/date-picker/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@twilio-paste/date-picker",
+  "version": "0.0.0",
+  "category": "user input",
+  "status": "production",
+  "description": "A form component used to select a date.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "date-fns": "2.21.3"
+  },
+  "peerDependencies": {
+    "@twilio-paste/input": "^3.0.6",
+    "@twilio-paste/style-props": "^3.0.1",
+    "@twilio-paste/styling-library": "^0.3.1",
+    "@twilio-paste/theme": "^5.0.1",
+    "@twilio-paste/types": "^3.1.1",
```

Comment:
Be sure to follow the peer dependency tree and list all sub peer dependencies in this package. Details can be found here https://github.com/twilio-labs/paste#peer-deps